### PR TITLE
update automating tests documentation

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

File: src/markdown-pages/docs/ecosystem/automating-overview.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+path: "/docs/ecosystem/automating/overview"
+title: "Automating tests"
+---
+Galasa SimBank shows how you can run a Galasa test locally inside an IDE. Running a test inside an IDE enables test and application development teams to run tests in a local workspace that interacts with real resources - ideal for minimising the risk of regressions from code changes.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/automating-overview.md:5]: updated locally

File: src/markdown-pages/docs/ecosystem/automating-overview.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+After writing a test to run locally on your machine, start to realise the power of Galasa by running that test in automation inside the [Galasa Ecosystem](/docs/ecosystem).
+
+To run a test in automation inside the Galasa Ecosystem, you need to complete the following high-level tasks:
+
+1. Structure a test to [run in automation](/docs/ecosystem/automating)
+2. [Deploy the test artifacts to a Maven repository](/docs/ecosystem/deploying-tests)
+3. Set up a new test stream
+4. Run the test headlessly as part of a pipeline
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/automating-overview.md:14]: updated locally

File: src/markdown-pages/docs/ecosystem/automating-overview.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Use this documentation to guide you through the process.
+
+### Running a test inside an IDE
+Compiling and running a test locally in Eclipse automatically creates a test jar file and an OSGi Bundle Repository (OBR) which enable Galasa to run any artifacts that are contained within the workspace.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/automating-overview.md:19]: updated locally

File: src/markdown-pages/docs/ecosystem/automating-overview.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+path: "/docs/ecosystem/automating/overview"
+title: "Automating tests"
+---
+Galasa SimBank shows how you can run a Galasa test locally inside an IDE. Running a test inside an IDE enables test and application development teams to run tests in a local workspace that interacts with real resources - ideal for minimising the risk of regressions from code changes.
+
+After writing a test to run locally on your machine, start to realise the power of Galasa by running that test in automation inside the [Galasa Ecosystem](/docs/ecosystem).
+
+To run a test in automation inside the Galasa Ecosystem, you need to complete the following high-level tasks:
+
+1. Structure a test to [run in automation](/docs/ecosystem/automating)
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/ecosystem/automating-overview.md:11]: update locally

**Review by hobbit1983 — APPROVED** [review]

**Comment by CaroMac** [comment]
This issue is for the review / update / approve part of the work. I have some questions / thoughts on these docs so I will use the review issue to go through these as it makes sense to do it as part of that piece of work. Links are here:
https://deploy-preview-402--galasa.netlify.app/docs/writing-own-tests
https://deploy-preview-402--galasa.netlify.app/docs/ecosystem/automating/overview
https://deploy-preview-402--galasa.netlify.app/docs/writing-own-tests/setting-up-galasa-project
https://deploy-preview-402--galasa.netlify.app/docs/writing-own-tests/setting-up-galasa-project-outside
https://deploy-preview-402--galasa.netlify.app/docs/ecosystem/automating/running-automation
https://deploy-preview-402--galasa.netlify.app/docs/ecosystem/automating/deploying-to-maven
https://deploy-preview-402--galasa.netlify.app/docs/writing-own-tests/test-streams

File: src/markdown-pages/docs/ecosystem/deploy-to-maven.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+title: "Deploying a test to a Maven repository"
+---
+
+To run a test in automation, the test engines need access to the test material. This is achieved by offering the test material as Maven artifacts, stored in a Maven repository. If you are using the Docker Operator to bring up an ecosystem, a Nexus repository is also instantiated, which can be used to store the artifacts. However, you can use any Maven repositories to which you have access.
+
+###  Define to Maven where the artifacts are to be deployed.
+This example uses a local Nexus repository that can be brought up locally on a workstation.
+Complete the following steps to define where you want to deploy the artifacts:
+1. Create or edit the ```settings.xml``` that is used by Maven. The default path of ```settings.xml``` is ```~/.m2```.
+    1. Update the details of the Nexus server for _id_, _username_, and _password_.
```
Comment (techcobweb) [inline src/markdown-pages/docs/ecosystem/deploy-to-maven.md:12]: "Refer to the maven documentation for details of how to encrypt this password if required"

**Pull request closed without merging**